components/layout/fragment.rs

```diff
@@ -425,18 +425,73 @@ impl ImageFragmentInfo {
         }
     }
 
+    pub fn tile_image_round(position: &mut Au,
+                            size: &mut Au,
+                            absolute_anchor_origin: Au,
+                            image_size: &mut Au) {
+        if *size == Au::new(0) || *image_size == Au::new(0) {
+            *position = Au::new(0);
+            *size =Au::new(0);
+            return;
+        }
+
+        let number_of_tiles = (size.to_f32_px() / image_size.to_f32_px()).round().max(1.0);
+        *image_size = *size / (number_of_tiles as i32);
+        ImageFragmentInfo::tile_image(position, size, absolute_anchor_origin, *image_size);
+    }
+
+    pub fn tile_image_spaced(position: &mut Au,
+                             size: &mut Au,
+                             tile_spacing: &mut Au,
+                             absolute_anchor_origin: Au,
+                             image_size: Au) {
+        if *size == Au::new(0) || image_size == Au::new(0) {
+            *position = Au::new(0);
+            *size =Au::new(0);
```

Comment:
nit: Minor formatting nit, there's a space missing here and in the previous function. Also, I think the usual way we write an empty `Au` value is `Au(0)` directly, though not sure if this has changed recently (please disregard it if that's the case).

Please fix these nits if any test fails, or otherwise in a followup, thanks!